src/utils/source.js

```diff
@@ -378,13 +380,19 @@ export function isLoading(source: Source) {
 }
 
 export function getTextAtPosition(source: ?Source, location: Location) {
-  if (!source || source.isWasm || !source.text) {
+  if (!source || !source.text) {
     return "";
   }
 
   const line = location.line;
   const column = location.column || 0;
 
+  if (source.isWasm) {
+    const { line: editorLine } = toEditorPosition(location);
+    const lines = renderWasmText(source.id, source.text);
+    return lines[editorLine];
+  }
+
```

Comment:
it'd be nice to have a unit test for this:

Maybe it is similar to this test:
https://github.com/devtools-html/debugger.html/blob/master/src/utils/tests/wasm.spec.js#L51

Reply:
lgtm, though renderWasmText is (very) expensive operation, it will be nice to cache renderWasmText result in case if several breakpoints are set.